tutorials/core-concepts/pieces/pieces-example.md

```diff
@@ -0,0 +1,242 @@
+
+## Create the Pieces Pages
+
+Since you created the new piece type with the `--pages` options, you already have the pieces pages module created. Now, you must register the pieces pages module in `app.js`, and create the templates for the index and show pages. 
+
+![](/.gitbook/assets/pieces-pages-directory.png) 
+
+1. Add the following code to `app.js` in the main modules section to register the pieces pages module.
+  
+  ```markup
+    modules { 
+      ... 
+      'foods-pages': { 
+          extend: 'apostrophe-pieces-pages'
+        }, 
+      ...
+    }
+  ```
+
+2. Then register the pieces pages as a new page type by adding this is the `pages` subsection of `modules`:
+
+  ```markup
+    {
+      name: 'foods-pages',
+      label: 'Food’'
+    }
+  ```
+
+3. Open `index.html` and add the following code:
+
+  ```markup
+    {% extends "layout.html" %}
+     {% block title %}{{ data.page.title }}
+    {% endblock %}
+
+    {% block main %}
+      <div class="main-content">
+        {% for piece in data.pieces %}
+          <h4><a href="{{ piece._url }}">{{ piece.title }}</a></h4> ${{ piece.price }}
+        {% endfor %}
+      </div>
+    {% endblock %} \
+  ```
+
+4. Open `show.html` and add the following code:
+
+
+  ```markup
+    {% extends "layout.html" %}
+    {% block title %}{{ data.piece.title }}{% endblock %}
+    
+    {% block main %}
+      <div class="main-content">
+        <h4> {{ data.piece.title }}
+        ${{ data.piece.price }}
+        
+        {{ apos.singleton( data.piece, 'image', 'apostrophe-images',
+          { edit: false }
+        )}}
+      </div>
+    {% endblock %}
+  ```
+
+### Displaying Pieces with Pieces Pages
+
+After you create and register the pieces pages module, you can create a new page for your piece type.
+
+1. While logged into Apostrophe, open the _Page Menu_ and select _New Page_.
+
+2. Under _Page Type_ select “Food”.
+
+3. Name the page “Menu” and click _Save_.
+
+![](/.gitbook/assets/pieces-create-pieces-page.png) 
+
+This creates the piece index page. The `index.html` template generates links for each piece of the piece type.
+
+![](/.gitbook/assets/pieces-menu-page.png) 
+
+The pages for each piece are generated from the `show.html`, so you now have individual pages for each piece.
+
+![](/.gitbook/assets/pieces-pieces-page.png) 
+
+The individual pieces pages will even integrate back with the widget, since now each piece has its own page at its own URL.
+
+## Next Steps
+
+From this example you can see how powerful pieces are, but we’ve only just scratched the surface. You can continue reading to try a [different piece example which uses more features, like joins](joins.md), or you can go to the [Advanced Pieces](/tutorials/advanced-development/advanced-pieces-topics) to learn more about concepts like [filters](/tutorials/advanced-development/advanced-pieces-topics/adding-filters.md) and [columns](/tutorials/advanced-development/advanced-pieces-topics/adding-columns.md).
```

Comment:
I don't think that first link to joins is well-motivated. Tell the user what they will get from that example.